packages/longhorn/longhorn/longhorn-1.8/generated-changes/overlay/values.yaml.orig

```diff
@@ -37,53 +37,53 @@ image:
       # -- Repository for the Longhorn Engine image.
```

Comment:
Can we make sure to cleanup the files generated by the `make patch` command, such as `packages/longhorn/longhorn-1.5/generated-changes/overlay/questions.yaml.orig` or `packages/longhorn/longhorn-1.5/charts/questions.yaml.orig`. (*.orig)

Ref the manual https://confluence.suse.com/display/LON/How+to+create+Rancher+Chart+PRs

Reply:
Checked and Removed. thanks